zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/DataSetInfo.scala

```diff
@@ -18,23 +19,84 @@ case class DataSetInfo(name: String,
                        dataInterval: Interval,
                        stats: Stats)
 
+case class UnresolvedSchema(typeName: String,
```

Comment:
Sure. I will add some documentations to this, and also JsonParser (which explains the workflow of parsing a json query).